Brokerages/GDAX/GDAXBrokerageFactory.cs

```diff
@@ -53,9 +55,10 @@ public override void Dispose()
         };
 
         /// <summary>
-        /// The brokerage model
+        /// The brokerage model. This uses the alorithm instance passed in <see cref="CreateBrokerage"/>
+        /// in order to properly populate the model with a reference to the algorithm's cashbook
         /// </summary>
-        public override IBrokerageModel BrokerageModel => new GDAXBrokerageModel();
+        public override IBrokerageModel BrokerageModel => new GDAXBrokerageModel(_algorithm?.Portfolio?.CashBook);
```

Comment:
This instance will be used in the event the user doesn't manually invoke `SetBrokerageModel` -- Since GDAX disabled margin, I think we should include `AccountType.Cash` here